backend/app/Http/Requests/UpdateUserHttpRequest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+final class UpdateUserHttpRequest extends FormRequest
+{
+    public function authorize(): bool
+    {
+        return true;
+    }
+
+    public function rules(): array
```

Comment:
it is fine, but could you also add "get" methods with type hintings it will prevent some bugs in the future:

public function getName() { return $this->get('name'); }